### PR TITLE
Allow container_t to call setxec on process

### LINKED ISSUE #55: Allow process:setexec for container_t
Hello,

While deploying openstack/tripleo on a SELinux enforcing CentOS 7, I get the following errors in the audit.log:
```
type=AVC msg=audit(1539849661.136:2404): avc:  denied  { setexec } for  pid=58245 comm="crond" scontext=system_u:system_r:container_t:s0:c80,c233 tcontext=system_u:system_r:container_t:s0:c80,c233 tclass=process
```

I've tried to understand how to inject the rule in the "container.te" of this repository, but it's way beyond my knowledge with selinux. Care to either explain me how to provide a good pull request, or do it?
The rule should look like that:
```
require {
	type container_t;
	class process setexec;
}

#============= container_t ==============
allow container_t self:process setexec;
```

Apparently there are "shortcuts" in the .te, and I don't want to mess it all ;).

Thank you!

### PR DESCRIPTION
**Pull request by cjeanner**
Closes #55

### CODE REVIEW

**Comment by rhatdan** [comment]
Also bump the version and VERSION file.
```
policy_module(container, 2.74.0)
```
sed 's|73|74|g' -i VERSION

File: container.te [hunk 1]
```diff
@@ -673,6 +673,8 @@ allow container_domain self:unix_stream_socket create_stream_socket_perms;
 dontaudit container_domain self:capability2  block_suspend ;
 allow container_domain self:unix_stream_socket { sendto create_stream_socket_perms };
 
+allow container_t self:process setexec;
```
[the remark this answers is not in the document]
Comment (cjeanner) [reply container.te]: ah, will do that a bit later, thanks! I wasn't sure about the use of the "container_domain" :).

**Comment by rhatdan** [comment]
Still need you to change the policy version in container.te to 74.


sed 's|73|74|g' -i container.te

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
Thanks @cjeanner

**Pull request merged**